src/software/new_geom/util/contains.cpp

```diff
@@ -1,6 +1,113 @@
+// TODO: rename `util` to algorithms
```

Comment:
TODO. EIther do it or link to an issue

Reply:
Wow, and I already had one made: https://github.com/UBC-Thunderbots/Software/issues/1494

But removing TODO since it's not super useful here.